.eslintrc.js

```diff
@@ -38,6 +39,7 @@ module.exports = {
       },
     ],
     semi: ['error', 'always'],
+    'react/react-in-jsx-scope': 'off',
```

Comment:
Looking at the doc for jsx-runtime, Im confused as to why `"react/jsx-uses-react": "off"` is not required here too, any idea?

Reply:
no clue